File: Java/Basics/file4.java

import java.util.*;

public class file4 {
    
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();
        int sum_rslt = 0, digit_value = 0;

        while(num != 0) {
            digit_value = num % 10;
            sum_rslt += digit_value;
            num /= 10;
        }

        System.out.println("Sum of its digit is: "+sum_rslt);

    }
}

File: C/Basics/file4.c

// find the sum of the digits of a number.......

#include<stdio.h>

int main()
{
    int num = 0,  sum_rslt = 0, digit_val = 0;
    
    scanf("%d", &num);

    while(num != 0) {
        digit_val = num % 10;
        sum_rslt += digit_val;  
        num /= 10;
    }
    printf("Sum of its digit is: %d\n", sum_rslt);
    return 0;
}
